src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -42,6 +42,7 @@ class SentryApplicationRow extends React.PureComponent {
       const query = Object.assign(qs.parse(url.query), installQuery);
       redirectUrl = `${url.protocol}//${url.host}${url.pathname}?${qs.stringify(query)}`;
       window.location.assign(redirectUrl);
+      return;
```

Comment:
accidental?

Reply:
No, for some reason the app redirects to another Sentry URL first, then redirects to the external URL. This is an attempt to fix that, but @MeredithAnya already merged a different fix that I need to rebase.